travvy/src/components/Home.vue

```diff
@@ -35,8 +35,9 @@
     <input type="text" class="travellers">
     <br>  
     <br>       
+    <center><button v-on:click="navigateTo({name:'AttractionsList'})" class="search"><span>Search </span></button></center>
+
```

Comment:
out of curiosity, why did you choose to remove the router-link and add the button?

Reply:
i was also wondering this because i ran into trouble when navigating from the different pages using a button object before but im glad you were able to figure it out

Reply:
Now the routing is handled by a function "navigateTo" that way it is easier to make routing dynamic in the future, and it allows the logic to lie outside the html code